apps/aetx/src/aetx.erl

```diff
@@ -168,11 +178,21 @@ accounts(#aetx{ cb = CB, tx = Tx }) ->
 signers(#aetx{ cb = CB, tx = Tx }, Trees) ->
     CB:signers(Tx, Trees).
 
+%% Internally we let 0 represent no TTL/infinity in order to keep the
+%% serialization as short as possible. Since there are no transactions in the
```

Comment:
Do you mean serialization for DB here? (as previous line says "internally" rather than "externally", that I would expect if this sentence refers to serialization for consensus)